great-idea-website/css/index.css

```diff
@@ -63,4 +63,157 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+nav{
+	width:90%;
+	margin: auto;
+	padding: 5px;
+}
+nav a{
+	display: inline-block;
+	padding:20px;
+	margin:20px;
+	text-decoration: none;
+	color: gray;
+	box-sizing: border-box;
+	text-align: : center;
+	font-size:16px;
```

Comment:
In this project, you can use `vertical-align` on the anchor elements to better achieve having the anchors and the image more centered. Moving forward, it's a much better idea to use Flexbox to center them.